plugin/wakatime.vim

```diff
@@ -150,17 +142,26 @@ let s:VERSION = '9.0.1'
         " First try install wakatime-cli in background, then using Vim's Python
         if !empty(python_bin)
             let install_script = s:plugin_root_folder . '/scripts/install_cli.py'
-            let cmd = [python_bin, '-W', 'ignore', install_script, s:home]
-            if s:IsWindows() && &shell =~ 'cmd'
-                let job_cmd = split(&shell) + split(&shellcmdflag) + cmd
-            else
-                let job_cmd = split(&shell) + split(&shellcmdflag) + [s:JoinArgs(cmd)]
+            if s:IsWindows() && s:Chomp(system(s:JoinArgs([python_bin, '-VV'])) !~ 'MSC')
```

Comment:
We can't use `system()` during init because it slows down startup for all Windows users.

Reply:
This should work instead?

https://github.com/wakatime/vim-wakatime/blob/bbfa0fb9a31fe812722468d7000a898bb83fc2fe/plugin/wakatime.vim#L93